<launch>
    <!-- Mask RCNN -->
    <node name="mask_rcnn" pkg="mask_rcnn_ros"
        type="mask_rcnn_service.py" output="screen">
        <remap from="input" to="left_gripper_sensor_d415_camera/color/image_raw" />
        <param name="visualization" value="true" />
        <param name="topic_and_srv" value="~detection" />
    </node>

    <!-- RVIZ -->
    <node name="rviz" pkg="rviz" type="rviz"
        args="-d $(find grasping_pipeline)/rviz/mask_rcnn_detection.rviz" />
</launch>
